### ISSUE TITLE
Reduce Language fingerprinting preference causes website to break

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#23918]
## Description 

Desktop Issue: #23306

Navigating to https://www.ticketweb.ca/ with Reduce Language fingerprinting preference on results in an empty page. With the pref turned off, page loads fine.

## Steps to Reproduce

   1. Go to https://www.ticketweb.ca/ with Reduce Language fingerprinting preference in brave://settings/shields turned on

## Actual result:

![image](https://user-images.githubusercontent.com/5284154/172459769-fc696393-368e-4ab3-9ada-f7ce3e93bea0.png)

## Expected result:
![image](https://user-images.githubusercontent.com/5284154/172459811-dc2f755d-a3c9-41c7-a9bb-3336489f27c3.png)

**Comment by srirambv** [comment]
Blocked on #23919 as the setting is yet to be implemented on Android

**Comment by pes10k** [comment]
I dont think this should still be broken on any platform.  @srirambv can you confirm this is still an issue?

**Comment by bsclifton** [comment]
Closing as stale. @srirambv please create a new issue if this is still a problem. Thanks!
